packages/twenty-front/src/modules/ui/input/editor/components/CustomAddBlockItem.tsx

```diff
@@ -0,0 +1,41 @@
+import { blockSchema } from '@/activities/blocks/schema';
+
+type CustomAddBlockItemProps = {
+  editor: typeof blockSchema.BlockNoteEditor;
+  children: React.ReactNode; // Adding the children prop
+};
+
+type ContentItem = {
+  type: string;
+  text: string;
+  styles: any;
+};
+export const CustomAddBlockItem = ({
+  editor,
+  children,
+}: CustomAddBlockItemProps) => {
+  const handleClick = () => {
+    const blockIdentifier = editor.getTextCursorPosition().block;
+    const currentBlockContent = blockIdentifier?.content as
+      | Array<ContentItem>
+      | undefined;
+
+    const [firstElement] = currentBlockContent || [];
+
+    if (firstElement === undefined) {
+      editor.openSelectionMenu('/');
+    } else {
+      editor.sideMenu.addBlock();
+      editor.openSelectionMenu('/');
+      editor.sideMenu.unfreezeMenu();
+    }
+  };
+  return (
+    <button
+      className="mantine-focus-auto m_99ac2aa1 mantine-Menu-item bn-menu-item m_87cf2631 mantine-UnstyledButton-root"
```

Comment:
Seems strange to have `m_99ac2aa1` or `m_87cf2631`, are you sure we need those?

Reply:
Will get back to you about this with some more clarity

Reply:
Hey @FelixMalfait,

I checked the classes m_99ac2aa1 and m_87cf2631 in the code. They’re handling specific styles like padding, cursor behavior, and other appearance settings using Mantine’s spacing and color variables.

If we want to avoid using these Mantine utility classes, I can replicate the styles manually. The only thing to consider would be how to handle Mantine’s spacing variables like `--mantine-spacing-xs` and `--mantine-spacing-sm` ourselves.

Let me know if you prefer that approach, or if you’re okay with keeping these classes for now.

![image](https://github.com/user-attachments/assets/26eabc47-3500-4b9f-a0c6-15b5a9d42bf3)